src/js/dom-elements.js

```diff
@@ -0,0 +1,30 @@
+export const mainEl = {
```

Comment:
This file should have DOM object elements that should provide methods or computed actions, no heavy logic should be involved in this file, which must be placed in a different file.